[2023-10-31T22:14:22.183+0100] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: trial_dag.hold_5_seconds manual__2023-10-31T21:14:12.469228+00:00 [queued]>
[2023-10-31T22:14:22.192+0100] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: trial_dag.hold_5_seconds manual__2023-10-31T21:14:12.469228+00:00 [queued]>
[2023-10-31T22:14:22.192+0100] {taskinstance.py:1361} INFO - Starting attempt 1 of 3
[2023-10-31T22:14:22.210+0100] {taskinstance.py:1382} INFO - Executing <Task(PythonOperator): hold_5_seconds> on 2023-10-31 21:14:12.469228+00:00
[2023-10-31T22:14:22.214+0100] {standard_task_runner.py:57} INFO - Started process 785 to run task
[2023-10-31T22:14:22.221+0100] {standard_task_runner.py:84} INFO - Running: ['airflow', 'tasks', 'run', 'trial_dag', 'hold_5_seconds', 'manual__2023-10-31T21:14:12.469228+00:00', '--job-id', '150', '--raw', '--subdir', 'DAGS_FOLDER/trial_dag.py', '--cfg-path', '/var/folders/3g/lt92y6v536s5sxynst_qrtsm0000gn/T/tmph7k_h0w2']
[2023-10-31T22:14:22.223+0100] {standard_task_runner.py:85} INFO - Job 150: Subtask hold_5_seconds
[2023-10-31T22:14:22.274+0100] {task_command.py:416} INFO - Running <TaskInstance: trial_dag.hold_5_seconds manual__2023-10-31T21:14:12.469228+00:00 [running]> on host qts-macbook-pro.local
[2023-10-31T22:14:22.356+0100] {taskinstance.py:1662} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='Brenda' AIRFLOW_CTX_DAG_ID='trial_dag' AIRFLOW_CTX_TASK_ID='hold_5_seconds' AIRFLOW_CTX_EXECUTION_DATE='2023-10-31T21:14:12.469228+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2023-10-31T21:14:12.469228+00:00'
[2023-10-31T22:14:27.357+0100] {python.py:194} INFO - Done. Returned value was: None
[2023-10-31T22:14:27.368+0100] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=trial_dag, task_id=hold_5_seconds, execution_date=20231031T211412, start_date=20231031T211422, end_date=20231031T211427
[2023-10-31T22:14:27.418+0100] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2023-10-31T22:14:27.435+0100] {taskinstance.py:2778} INFO - 0 downstream tasks scheduled from follow-on schedule check
